### DISCUSSION TITLE
v2021.11.19

### DISCUSSION THREAD

**Discussion by slawekjaranowski** [s4u/pgp-keys-map#429] [Announcements]
# What's Changed

## :fire: New features

- Adding my public key for mustache.java #381
- New key for net.sf.saxon #391
- New key for org.codehaus #392
- Added mapping for xmlbeans-5.0.2 #393
- New key for io.vavr #402
- New key for org.junit-pioneer #403
- New key for commons-cli #406
- Added mapping for poi-5.1.0 #411

## :toolbox: Dependency updates

- Bump stripe-java from 20.82.0 to 20.83.0 in /pgp-keys-map-test1 #380
- Bump spring-core.version from 5.3.10 to 5.3.11 in /org-springframework-test #378
- Bump parent from 2.13.0 to 2.15.0 #382
- Bump semanticcms-parent from 1.13.2 to 1.13.4 in /pgp-keys-map-test1 #383
- Bump pragmatickm-parent from 1.11.2 to 1.11.3 in /pgp-keys-map-test1 #384
- Bump stripe-java from 20.83.0 to 20.84.0 in /pgp-keys-map-test1 #385
- Bump ao-oss-parent from 1.17.2 to 1.17.4 in /pgp-keys-map-test1 #387
- Bump nmw-oss-parent from 1.14.2 to 1.14.3 in /pgp-keys-map-test1 #388
- Bump postgresql from 42.2.24 to 42.3.0 in /pgp-keys-map-test1 #389
- Bump mysql-connector-java from 8.0.26 to 8.0.27 in /pgp-keys-map-test1 #390
- Bump spotbugs-maven-plugin from 4.4.1 to 4.4.2.1 in /pgp-keys-map-test1 #386
- Bump google-java-format from 1.11.0 to 1.12.0 in /pgp-keys-map-test1 #395
- Bump javatator from 0.5.0 to 0.5.1 in /pgp-keys-map-test1 #396
- Bump protobuf-java from 3.18.1 to 3.19.0 in /pgp-keys-map-test1 #397
- Bump stripe-java from 20.84.0 to 20.85.0 in /pgp-keys-map-test1 #398
- Bump spring-boot.version from 2.5.5 to 2.5.6 in /org-springframework-test #400
- Bump spotbugs-maven-plugin from 4.4.2.1 to 4.4.2.2 in /pgp-keys-map-test1 #401
- Bump spring-core.version from 5.3.11 to 5.3.12 in /org-springframework-test #399
- Bump protobuf-java from 3.19.0 to 3.19.1 in /pgp-keys-map-test1 #405
- Bump commons-cli from 1.4 to 1.5.0 in /pgp-keys-map-test1 #404
- Bump postgresql from 42.3.0 to 42.3.1 in /pgp-keys-map-test1 #407
- Bump stripe-java from 20.85.0 to 20.86.0 in /pgp-keys-map-test1 #408
- Bump xmlsec from 2.2.3 to 2.3.0 in /pgp-keys-map-test1 #410
- Bump stripe-java from 20.86.0 to 20.86.1 in /pgp-keys-map-test1 #413
- Bump error_prone_annotations from 2.9.0 to 2.10.0 in /pgp-keys-map-test1 #412
- Bump jquery-ui from 1.12.1 to 1.13.0 in /pgp-keys-map-test1 #414
- Bump jna from 5.9.0 to 5.10.0 in /pgp-keys-map-test1 #416
- Bump spring-core.version from 5.3.12 to 5.3.13 in /org-springframework-test #417
- Bump junit-pioneer from 1.4.2 to 1.5.0 in /pgp-keys-map-test1 #420
- Bump stripe-java from 20.86.1 to 20.87.0 in /pgp-keys-map-test1 #419
- Bump qdox from 2.0.0 to 2.0.1 in /pgp-keys-map-test1 #418
- Bump tomcat-api from 10.1.0-M6 to 10.1.0-M7 in /pgp-keys-map-test1 #421
- Bump struts2-core from 2.5.26 to 2.5.27 in /pgp-keys-map-test1 #422
- Bump tomcat-embed-core from 10.1.0-M6 to 10.1.0-M7 in /pgp-keys-map-test1 #423
- Bump stripe-java from 20.87.0 to 20.88.0 in /pgp-keys-map-test1 #424
- Bump stripe-java from 20.88.0 to 20.89.0 in /pgp-keys-map-test1 #425
- Bump spring-boot.version from 2.5.6 to 2.5.7 in /org-springframework-test #426
- Bump spotbugs-maven-plugin from 4.4.2.2 to 4.5.0.0 in /pgp-keys-map-test1 #427
- Bump pgpverify-maven-plugin from 1.14.1 to 1.15.0 #428

# :heart: Thanks
Many thanks for collaboration on this release for: @ao-apps, @slawekjaranowski and @spullara

<hr /><em>This discussion was created from the release <a href='https://github.com/s4u/pgp-keys-map/releases/tag/v2021.11.19'>v2021.11.19</a>.</em>

**Comment by ao-apps** [comment]
Thank you
